unityfiles/Assets/Scripts/Game/HealthController.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class HealthController : MonoBehaviour
+{
+    [Tooltip("the ammont of heth a given entety has.")]
+    [SerializeField] 
+    private float entityHealth;
+
+    // -- properties
+
+    public float EntityHealth
+    {   
+        get{return this.entityHealth;} 
+        set{this.entityHealth = value;}
+    }
+
+   
+
+    // Update is called once per frame
+    void Update()
+    {
+        if (this.entityHealth <= 0f)

```

Comment:
Legg til commentar `// magic number is health. eg. player is dead`

Reply:
helth < 0 er selvsigende